server/datastore/mysql/software_installers_test.go

```diff
@@ -2085,327 +2193,98 @@ Software won't be installed on Linux hosts with Debian-based distributions becau
 	require.Equal(t, "[Install software] Something2 (msi) 3", team3Policies[2].Name)
 }
 
-// TODO(uniq): This is intended to be a temporary test for happy path testing of the new
-// unified queue. It likely can be deleted assuming that existing tests that are currently failing
-// are updated once the full unified queue is implemented.
```

Comment:
Removed as per the comment, this should all be covered via other tests and now all Datastore tests pass.